src/ledger/database/hashmap.zig

```diff
@@ -351,12 +364,12 @@ pub fn SharedHashMapDB(comptime column_families: []const ColumnFamily) type {
 
                 pub fn nextKey(self: *@This()) anyerror!?cf.Key {
                     const index = self.nextIndex() orelse return null;
-                    return key_serializer.deserialize(cf.Key, self.allocator, self.keys[index]);
+                    return try key_serializer.deserialize(cf.Key, self.allocator, self.keys[index]);
```

Comment:
Given that the value gotten from `try` is not used within the function but returned immediately, is there a functional difference/advantage over how it was before without the `try`

Reply:
Without try, this won't compile:
```zig
src/ledger/database/hashmap.zig:367:54: error: expected type 'anyerror!?ledger.schema.schema.address_signatures__struct_16305', found '@typeInfo(@typeInfo(@TypeOf(ledger.database.interface.serializer(.big).deserialize__anon_50347)).Fn.return_type.?).ErrorUnion.error_set!ledger.schema.schema.address_signatures__struct_16305'
                    return key_serializer.deserialize(cf.Key, self.allocator, self.keys[index]);
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/ledger/database/hashmap.zig:367:54: note: error union payload 'ledger.schema.schema.address_signatures__struct_16305' cannot cast into error union payload '?ledger.schema.schema.address_signatures__struct_16305'
src/ledger/schema.zig:87:16: note: struct declared here
        .Key = struct {
               ^~~~~~
src/ledger/database/hashmap.zig:365:56: note: function return type declared here
                pub fn nextKey(self: *@This()) anyerror!?cf.Key {
                                               ~~~~~~~~^~~~~~~~
```

Before this PR, this code was unused.